.github/ISSUE_TEMPLATE/DOCUMENTATION.md

```diff
@@ -0,0 +1,6 @@
+---
+name: 📃 Documentation
+about: Report an issue related to documentation
+title: 'Documentation: '
+labels: 'documentation'
+---
```

Comment:
Will further fields be added here? Similar to the bug report? With documentation we may want various categories for genuine technical errors vs say, grammar errors

Reply:
@JamesACS do you have something in mind? I tried to find some examples, but I didn't find anything that I liked